MQ2PluginManager.cpp

```diff
@@ -22,6 +22,7 @@ static void PluginManagerCommand(PlayerClient*, const char*);
 static bool s_showWindow = true;
 static bool s_showWinOutOfGame = false;
 static void ImGui_ToggleWindow();
+static char s_searchPlugin[24] = { 0 };
```

Comment:
```suggestion
static char s_searchPlugin[64] = { 0 };
```

this should be larger.